lib/ansible/modules/network/avi/avi_cloud.py

```diff
@@ -30,70 +30,93 @@
         type: bool
     autoscale_polling_interval:
         description:
-            - Cloudconnector polling interval for external autoscale groups.
+            - Cloudconnector polling interval in seconds for external autoscale groups, minimum 60 seconds.
+            - Allowed values are 60-3600.
             - Field introduced in 18.2.2.
             - Default value when not specified in API or module is interpreted by Avi Controller as 60.
         version_added: "2.9"
+        type: int
     aws_configuration:
         description:
             - Awsconfiguration settings for cloud.
+        type: dict
     azure_configuration:
         description:
             - Field introduced in 17.2.1.
         version_added: "2.5"
+        type: dict
     cloudstack_configuration:
         description:
             - Cloudstackconfiguration settings for cloud.
+        type: dict
     custom_tags:
         description:
             - Custom tags for all avi created resources in the cloud infrastructure.
             - Field introduced in 17.1.5.
         version_added: "2.5"
+        type: list
     dhcp_enabled:
         description:
             - Select the ip address management scheme.
             - Default value when not specified in API or module is interpreted by Avi Controller as False.
         type: bool
     dns_provider_ref:
         description:
-            - Dns profile for the cloud.
+            - DNS Profile for the cloud.
             - It is a reference to an object of type ipamdnsproviderprofile.
+        type: str
+    dns_resolution_on_se:
+        description:
+            - By default, pool member fqdns are resolved on the controller.
+            - When this is set, pool member fqdns are instead resolved on service engines in this cloud.
+            - This is useful in scenarios where pool member fqdns can only be resolved from service engines and not from the controller.
+            - Field introduced in 18.2.6.
+            - Default value when not specified in API or module is interpreted by Avi Controller as False.
+        version_added: "2.9"
+        type: bool
     docker_configuration:
         description:
             - Dockerconfiguration settings for cloud.
+        type: dict
     east_west_dns_provider_ref:
         description:
-            - Dns profile for east-west services.
+            - DNS Profile for East-West services.
             - It is a reference to an object of type ipamdnsproviderprofile.
+        type: str
     east_west_ipam_provider_ref:
         description:
             - Ipam profile for east-west services.
             - Warning - please use virtual subnets in this ipam profile that do not conflict with the underlay networks or any overlay networks in the cluster.
-            - For example in aws and gcp, 169.254.0.0/16 is used for storing instance metadata.
+            - For example in aws and gcp, 169.254.0. 0/16 is used for storing instance metadata.
```

Comment:
Why did you introduce a space here? The original expression looks better to me.
```suggestion
            - For example in AWS and GCP, 169.254.0.0/16 is used for storing instance metadata.
```